.env

```
@@ -0,0 +1,47 @@
+# Local environment variables for OpenLegislation (DO NOT commit secrets)
+# Copy to `.env` and edit values for your machine.
+# Sample values are placeholders — replace before use.
+
+# Postgres connection - CBWDELLR720 Homelab Server
+PGHOST=172.28.82.205
+PGPORT=5432
+PGUSER=opendiscourse
+PGPASSWORD=opendiscourse123
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The PGPASSWORD key should go before the PGPORT key

(UnorderedKey)